Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -265,20 +265,20 @@ private void PrintTaskRequested(PrintManager sender, PrintTaskRequestedEventArgs
                         switch (args.Completion)
                         {
                             case PrintTaskCompletion.Failed:
-                                {
-                                    OnPrintFailed?.Invoke();
-                                    break;
-                                }
+                            {
```

Comment:
this was unintentional. is the old indentation preferred?

Reply:
I don't think we have a rule for this. I personally don't like braces for case blocks. Are they required here? If not, just remove them.